dev/App.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component } from "solid-js";
+import toast, { Toaster } from "../src";
+
+const App: Component = () => {
+  let toastID: string[] = [];
+  const popSuccess = () =>
+    toastID.push(toast.success("Success!", { duration: Infinity }));
+  const popError = () =>
+    toastID.push(toast.error("Error!", { duration: Infinity }));
+  const closeAll = () => {
+    toastID.forEach((id) => toast.dismiss(id));
+    toastID = [];
+  };
```

Comment:
Hey there! This is great. Yeah having a dev environment will be very helpful to do some quick mockups. I have some changes to suggest. This can just be `toast.dismiss()` If you dont provide any toastID then it closes all the toasts by default :D So We will also not need to keep track of all the toastIDs in the array
```suggestion
const App: Component = () => {
  const popSuccess = () =>
    toast.success("Success!", { duration: Infinity });
  const popError = () =>
    toast.error("Error!", { duration: Infinity });
  const closeAll = () => {
   toast.dismiss();
  };
```